122.py

```diff
@@ -0,0 +1,9 @@
+class Solution:
+    def maxProfit(self, prices: List[int]) -> int:
+        max_profit = 0
```

Comment:
There is no comparison involved throughout this function body. I don't think it's appropriate to name it `max_profit` (is it a maximum of what?). It's just the total profit you can make by deploying some strategy.